src/mlpack/methods/ann/loss_functions/dice_loss.hpp

```diff
@@ -56,7 +51,7 @@ class DiceLoss
   /**
    * Create the DiceLoss object.
    *
-   * @param smooth The parameter to avoid overfitting.
+   * @param smooth The laplace smoothing parameter.
```

Comment:
Should this be `Laplace`?  I know that is really pedantic though. :)

Reply:
Nice catch!